aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```
@@ -458,6 +460,47 @@ Resources:
     DependsOn:
       - FargateServiceTaskDefTaskRole
 
+  ECSServiceScalableTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    Properties:
+      MaxCapacity: 20 # Upper limit for scaling, can be increased as needed
+      MinCapacity: 3 # High availability minimum
+      ResourceId: !Join
+        - '/'
+        - - "service"
+          - !Ref EcsCluster
+          - !GetAtt FargateService.Name
```

IMO it's more readable with `!Sub`

```suggestion
      ResourceId: !Sub "service/${EcsCluster}/${FargateService.Name}"
```

([docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html#w27aac23c24c67b7))